ibis/backends/hdf5/tests/test_hdf5.py

```diff
@@ -3,11 +3,8 @@
 from pandas.util import testing as tm
 
 import ibis
-from ibis.file.client import FileDatabase
-
-pytest.importorskip('tables')  # isort:skip
```

Comment:
is this not needed anymore because we are *only* running this in the specified env (which by-definition has this)?

if someone runs locally this would fail then. i guess that is fine. we just need to be consistent about that (e.g. running locally should i be expected to install *everything*? prob not.

Reply:
This is correct. What we are expected to do now to test locally is the same as in the CI, and it's explained in the contributing docs:
```sh
PYTEST_BACKENDS="hdf5 parquet" ./ci/run_tests.sh
```
I think this is much better than the skip approach, since tests will fail if a backend is specified but its dependencies are not installed (as opposed as failing silently, which caused problems in the CI often).

I've been using that way of testing locally for a while now, and I'm quite happy with it. I can test the backend I'm changing only, or just `pandas sqlite` if I don't want to have a container running.

Reply:
kk all makes sense, sounds totally fine. yeah de-facto the backends are separate so we don't expect to install them together (though note that the conda recipe itself does this)